slacky/main.py

```diff
@@ -125,6 +125,7 @@ class Slacky:
     container_publishes: dict = {}
     last_interval_check: datetime = datetime.now()
     do_save_state: bool = False
+    state_file_path = os.environ.get("STATE_FILE_PATH", "state/state.pickle")
```

Comment:
the default should be the previous default, so

```suggestion
    state_file_path = os.environ.get('STATE_FILE_PATH', Path(__file__).resolve().parent / 'state.pickle')
```

Reply:
fixed